src/sagemaker/mxnet/README.rst

```diff
@@ -666,10 +679,29 @@ The ``output_fn`` has the following signature:
     def output_fn(prediction, content_type)
 
 Where ``prediction`` is the result of invoking ``predict_fn`` and
-``content_type`` is the InvokeEndpoint requested response content-type. The function should return a byte array of data serialized to content_type.
+``content_type`` is the InvokeEndpoint requested response content type. The function should return a byte array of data serialized to the expected content type.
```

Comment:
"...is the requested response content type for ``InvokeEndpoint``. The function should return an array of bytes serialized to the expected content type."